src/storm/storage/expressions/ToCppVisitor.cpp

```diff
@@ -151,6 +151,8 @@ boost::any ToCppVisitor::visit(BinaryNumericalFunctionExpression const& expressi
             expression.getSecondOperand()->accept(*this, data);
             stream << ")";
             break;
+        case BinaryNumericalFunctionExpression::OperatorType::Logarithm:
+            STORM_LOG_THROW(false, storm::exceptions::NotSupportedException, "Log expressions not implemented for C++ translation.");
```

Comment:
Could we not use something like `std::log2(first) / std::log2(second)` here? But might also not be relevant anymore now that JIT support was dropped.

Reply:
Yeah I did not include this for now because we dropped jit and it didn't seem relevant. If it becomes relevant at some point, it's not too hard to add, I guess.
`std::log2` requires `#include <cmath>` and I don't know if this can be assumed.

Reply:
I agree that we can always add it if needed.